2019-11-14 06:24:20.101 - [Info] MITM Version: 1.26
2019-11-14 06:24:20.107 - [Info] Auto Access Enabled: true
2019-11-14 06:24:20.107 - [Debug] [Init] Auto Access Barrier: true
2019-11-14 06:24:20.130 - [Info] SSH man-in-the-middle server for 172.20.0.5 listening on 0.0.0.0:10104
2019-11-14 06:32:43.064 - [Debug] [Connection] Attacker connected: 140.143.242.159 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-14 06:32:44.568 - [Debug] [Auth] Attacker 140.143.242.159 trying to authenticate with "password"
2019-11-14 06:32:44.571 - [Debug] [Auto Access] Attacker: 140.143.242.159, Threshold: 1, Attempts: 1
2019-11-14 06:32:44.571 - [Debug] [Auto Access] Compromising the honeypot
2019-11-14 06:32:44.571 - [Debug] [Auto Access] Adding the following credentials: "daemon:nobody4444"
2019-11-14 06:32:44.864 - [Debug] [Auth] Valid credentials - Password Authentication
2019-11-14 06:32:44.865 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.5
2019-11-14 06:32:44.983 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2019-11-14 06:32:45.954 - [Debug] [Connection] Attacker connected: 140.143.242.159 | Client Identification: SSH-2.0-libssh-0.6.3
2019-11-14 06:32:47.437 - [Debug] [Auth] Attacker 140.143.242.159 trying to authenticate with "password"
2019-11-14 06:32:47.438 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2019-11-14 06:32:47.440 - [Debug] [Auth] Attacker: 140.143.242.159 has so far made 1 attempts. Remaining: 5 attempts
2019-11-14 06:32:47.951 - [Debug] [Connection] Attacker closed the connection
2019-11-14 06:32:48.425 - [Debug] [EXEC] Noninteractive mode attacker command: cat /proc/cpuinfo | grep name | wc -l
2019-11-14 06:32:48.989 - [Debug] [Connection] Attacker closed connection
2019-11-14 06:32:48.996 - [Debug] [LXC] Container's OpenSSH server ended connection
2019-11-14 06:32:48.997 - [Debug] [LXC] Container's OpenSSH server closed connection
